examples/platforms/efr32/radio.c

```diff
@@ -844,10 +843,24 @@ void RAILCb_FreeMemory(void *aHandle)
     (void)aHandle;
 }
 
-void otPlatRadioSetDefaultTxPower(otInstance *aInstance, int8_t aPower)
+otError otPlatRadioGetTransmitPower(otInstance *aInstance, int8_t *aPower)
+{
+    otError error = OT_ERROR_NONE;
+    (void)aInstance;
+
+    otEXPECT_ACTION(aPower != NULL, error = OT_ERROR_INVALID_ARGS);
+    *aPower = RAIL_TxPowerGet(aPower);
```

Comment:
notice that RAIL_TxPowerGet() function declaration defined in rail.h as below:
`int32_t RAIL_TxPowerGet(void)`, remove func parameter `aPower`here?